src/Cli/dotnet/commands/dotnet-workload/install/xlf/LocalizableStrings.zh-Hans.xlf

```diff
@@ -242,6 +242,16 @@
         <target state="new">Workload installation failed: {0}</target>
         <note />
       </trans-unit>
+      <trans-unit id="WorkloadNotRecognized">
+        <source>Workload with id {0} is not recognized.</source>
+        <target state="new">Workload with id {0} is not recognized.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="WorkloadNotSupportedOnPlatform">
+        <source>Workload with id {0} is not supported on this platform.</source>
```

Comment:
Do we have a value for "this platform" that can maybe be included in in the message?

Reply:
There's not an easy way to get the current platform but we should provide the supported platforms? Would this be helpful to users, as it would be a list of rids?

Reply:
I'm always up for providing people with more information, but not if the list is like 20 items as that might be overwhelming.

Reply:
Right now I'm only seeing max 5-ish, but that was my concern with including it- the list could get pretty big